src/solver/CMakeLists.txt

```diff
@@ -73,8 +73,13 @@ add_executable(antares-solver
 
 set_target_properties(antares-solver PROPERTIES OUTPUT_NAME ${exec_name})
 
-set(ANTARES_SOLVER_LIBS		
-	date
+set(ANTARES_SOLVER_LIBS
+	Antares::date
+	Antares::benchmarking
+	Antares::result_writer
+	Antares::sys
+	Antares::infoCollection
+	Antares::checks

```

Comment:
We should avoid heterogeneous naming in CMake targets

- infoCollection
- result_writer
- tests-ts-numbers

In other words, we should pick a convention and stick to it.

Reply:
See #1606 for discussion